src/pom.xml

```diff
@@ -844,8 +844,18 @@
    </dependency>
    <dependency>
      <groupId>org.slf4j</groupId>
+     <artifactId>slf4j-api</artifactId>
+     <version>1.7.14</version>
```

Comment:
Thanks for adding to dependency management section.

(I agree with only two entries it is not worth breaking out a ``${slfj.version}`` property.